lib/stripe/subscription_item.rb

```diff
@@ -9,6 +9,8 @@ class SubscriptionItem < APIResource
 
     OBJECT_NAME = "subscription_item".freeze
 
+    custom_method :usage_record_summaries, http_verb: :get
+
```

Comment:
I think this one is a mistake in the approach. This is the "old way" where you use `persons` on an existing account. Shouldn't it have `list_usage_records` instead?

Reply:
Good point. We could use `nested_resource_class_methods` instead, but the proper way to do that would be to use `:usage_record_summary` as the nested resource name and that would define the list method as `list_usage_record_summarys` because the DSL is not smart enough to know that the plural of "summary" is "summaries" and not "summarys". I'll see about adding a way to override the methods' names, or we might just want to define the method manually.

In any case, let's remove the `custom_method` call from this PR and fix separately.

Reply:
removed